src/Components/SiteColorsPicker.scss

```diff
@@ -1 +1,70 @@
 @import 'react-color-palette/css';
+
+.mt-2 {
+  margin-top: 0.5rem !important;
+}
```

Comment:
* [ ] We should put global styles in global CSS (`scrivitoExtensions.scss`) instead of component-specific CSS. (Or make the selector component specific.)